doc/guides/getting_started/src/e2e_regression_setup.md

```diff
@@ -0,0 +1,40 @@
+# ROM E2E Regressions setup
+This guide will help you to setup an environment to run the ROM E2E tests in an Ubuntu 20.04 machine.
+
+## Hardware setup
+### FPGA
+The E2E ROM tests run in the CW310 FPGA board with the following configuration:
+ - The power supply connected to the J11 connector.
+ - A USB cable connecting the Host PC to the USB-C Data (J8) connector. 
+ - The USRDIP#0 (SW4 header) is set to 0.
+ - The rest of the USRDIPs are set to 1.
+ - The S3 switch (next to USB connector J9) is set to USB.
+ - The S2 switch (next to JTAG header) is set to +3.3V.
+ - Optionally, a USB cable connecting the host PC to the USB-C J9 connector for future tests that will test the USB block.
+  
+### JTAG adapter
+Some tests will use the JTAG interface to interact with Opentitan, as the CW310 doesn't have a embedded JTAG adapter, we can use an external adapter (Olimex ARM-USB-TINY-H JTAG) connected to the J13 header in the CW310.
+
+### Full hardware setup
+![Full hardware setup](CW310_setup.png)
+
+## Software setup 
+The software dependencies are covered by other guides in this book and are linked below.
+- Clone the Opentitan repository and install the software dependencies: [Get started](README.md).
+- Setup the FPGA: [FPGA guide](./setup_fpga.md).
+- Install Vivado to be able to splice bitstreams: [Installing Vivado](install_vivado/README.md).
+
+**Note**: Make sure that you added the udev rules for the [FPGA](install_vivado/README.md#device-permissions-udev-rules) board and the [JTAG](setup_fpga.md#device-permissions-udev-rules) adapter.
+
+## Running the ROM E2E tests
+To run the ROM E2E tests you just need to run the command `bazel test` pointing at the `sw/device/silicon_creator/rom/e2e` directory.
+```console
+cd ${REPO_TOP}
+./bazelisk.sh test --define DISABLE_VERILATOR_BUILD=true --define bitstream=vivado --test_tag_filters=-verilator,-dv,-broken --build_tests_only //sw/device/silicon_creator/rom/e2e/...
+```
+This bazel command will build a bitstream, splice it with the different OPT configurations, load the FPGA and run all the ROM_E2E tests available under `sw/device/silicon_creator/rom/e2e`.
+If you want to save time you can use cached bitstream by changing the `bitstream=vivado` to `bitstream=gcp_splice`.
```

Comment:
Does using `bitstream=gcp_splice` take the ROM version associated with the repo `HEAD`? If not, it is work mentioning here in an additional note.

Reply:
I believe so, can you confirm it @alphan ?